#used for xsens_driver

Header header

# GPS status [true if detected]
bool status

# Latitude [degrees]. Positive is north of equator; negative is south.
float64 latitude
# Longitude [degrees]. Positive is east of prime meridian; negative is west.
float64 longitude

# displacement from a reference point (assumed to be a home location)
float64 X
float64 Y
